coordinator-client/src/auth-dummy.ts

```diff
@@ -16,4 +16,10 @@ export class AuthDummy implements Auth {
     }): string {
         return `Dummy ${this.participantId}`
     }
+
+    signMessage(
+        message: string, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
FYI (I know this has been merged already): Another way to avoid this warning without disabling the rule is by starting the name with an underscore (`_message`).